ark/utils/spatial_analysis_utils.py

```diff
@@ -163,6 +165,10 @@ def compute_close_cell_num(dist_mat, dist_lim, analysis_type,
 
     cell_labels = []
 
+    # Assign column names for subsetting (cell labels and cell type ids)
+    cell_label_col = settings.CELL_LABEL
```

Comment:
What are these used for?

Reply:
aren't they used in the `get_pos_cell_labels_` functions?

Reply:
Ah, I didn't see any diff adding them in, but that's because they were already defined in the function args. Can you set the defaults there?